ct/komga.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+source <(curl -s https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/misc/build.func)
+# Copyright (c) 2021-2024 community-scripts ORG
+# Author: madelyn (DysfunctionalProgramming)
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+function header_info {
+clear
+cat <<"EOF"
+ _  __                           
+| |/ /___  _ __ ___   __ _  __ _ 
+| ' // _ \| '_ ` _ \ / _` |/ _` |
+| . \ (_) | | | | | | (_| | (_| |
+|_|\_\___/|_| |_| |_|\__, |\__,_|
+                     |___/       
+EOF
+}
+header_info
+check_container_storage
+check_container_resources
```

Comment:
```suggestion
```

Reply:
Why? Its the new function

Reply:
It is a duplicate line and in the wrong place, it should only be in update function